sxt/multiexp/pippenger2/in_memory_partition_table_accessor_utility.t.cc

```diff
@@ -43,7 +43,7 @@ TEST_CASE("we can create a partition table accessor from given generators") {
   memmg::managed_array<E> table(table_dev.size());
 
   SECTION("we can create an accessor from a single generators") {
-    auto accessor = make_in_memory_partition_table_accessor<E>(basct::subspan(generators, 0, 1));
+    auto accessor = make_in_memory_partition_table_accessor<E, E>(basct::subspan(generators, 0, 1));
```

Comment:
If you added a default template parameter, `template <bascrv::element T, class U = T>`, could you avoid needing to use `<E, E>` in cases where the accessor to partition data sums and the MSM result are the same type?

Reply:
I made an adjustment that lets allows you to call the function with only a single template argument if the types are the same.